packages/gatsby-docs/src/pages/setup/customer_setup.mdx

```diff
@@ -2,20 +2,26 @@
 title: Setting up the tools
 ---
 
-> **Apologies, the UI Upgrade Helper is temporarily unavailable during a transition from IBM hosting services.**
+> **Apologies, the UI Upgrade Helper is temporarily unavailable for download during a transition from IBM hosting services.**
```

Comment:
The images are still up currently and latest information is that they will be taken down in 2 weeks approximately.. Can we say something like "the UI Upgrade Helper will soon be temporarily unavailable for download during a transition from IBM hosting services"  or something like that?

Reply:
Done